gui/launcher.h

```diff
@@ -23,14 +23,42 @@
 #ifndef GUI_LAUNCHER_DIALOG_H
 #define GUI_LAUNCHER_DIALOG_H
 
+// Disable the grid for platforms that disable fancy themes
+#ifdef DISABLE_FANCY_THEMES
+#define DISABLE_LAUNCHERDISPLAY_GRID
+#endif
+#define kSwitchLauncherDialog -2
+
 #include "gui/dialog.h"
 #include "engines/game.h"
 
+using Common::String;
+using Common::U32String;
+using Common::Array;
+
+typedef Array<String> StringArray;
+typedef Array<U32String> U32StringArray;
+
 namespace GUI {
 
+#ifndef DISABLE_LAUNCHERDISPLAY_GRID
+enum LauncherDisplayType {
+	kLauncherDisplayList = 1,
+	kLauncherDisplayGrid = 2
+};
+
+enum {
```

Comment:
Seems unused.

Reply:
This enum is used in LauncherDialog::handleCommand() to toggle between list and grid view.